lib/rubycritic/commands/base.rb

```diff
@@ -9,7 +9,7 @@ def initialize(options)
       end
 
       def execute
-        fail NotImplementedError
+        raise NotImplementedError
```

Comment:
>> These are quite recent changes but it suggests to move away from fail - Rubocop 0.37.0 does just that.

Yep, exactly :laughing: 

>> Personal preferences are over-rated

I assume @lucasmazza agrees with that, so let's get this merged ;)